boards/native/board.c

```diff
@@ -0,0 +1,72 @@
+/**
+ * Native Board board implementation
+ *
+ * Copyright (C) 2018 Gilles DOFFE <g.doffe@gmail.com>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ *
+ * @ingroup boards_native
+ * @{
+ * @file
+ * @author  Gilles DOFFE <g.doffe@gmail.com>
+ * @}
+ */
+#include <inttypes.h>
+#include <stdio.h>
+
+/* RIOT includes */
+#include <board.h>
+#include <log.h>
+
+#ifdef QDEC_NUMOF
+
+#define SIMU_ENC_BUFSIZE    3
+extern int32_t qdecs_value[QDEC_NUMOF];
+
+void native_motor_driver_qdec_simulation(
```

Comment:
It simulates independant motor feedback (through QDEC) with an average. Just an example.
If the quadrature feedback is mechanicly coupled with motor, it could just be `qdec_value = pwm`.